cartographer/mapping/map_builder.cc

```diff
@@ -237,6 +250,12 @@ void MapBuilder::LoadMap(io::ProtoStreamReader* const reader) {
                                    proto.submap().submap_id().submap_index()});
       sparse_pose_graph_->AddSubmapFromProto(submap_pose, proto.submap());
     }
+    if (proto.has_imu_data()) {
```

Comment:
No code in comments, please. At best this adds to the maintenance burden, at worst it gets out of sync. Remove here and add in the appropriate follow up PR?